server/models/models.js

```diff
@@ -5,9 +5,22 @@
 'use strict';
 
 var pg = require('pg');
+var es = require('elasticsearch');
 
 var config = require('../config');
-var log = require('../logger')('server.db.db');
+var log = require('../logger')('server.models');
+
+// >:-(
+// https://github.com/elasticsearch/elasticsearch-js/issues/33
+var esParamsFactory = function() {
+  return {
+    host: {
+      host: config.get('db.elasticsearch.host'),
+      post: config.get('db.elasticsearch.port'),
+      log: 'verbose'
```

Comment:
short answer: this is one of the todos that I added to #91.

longer answer: the elasticsearch people implemented their own logger, which is Bunyan compatible, but which uses 5 loglevels instead of mozlog's 7. They also use "warning" instead of "warn". So, we'll actually need some friggin mapper between the two loggers, or we'll get rid of mozlog. TBD.